mdoc/src/main/scala/mdoc/PostModifier.scala

```diff
@@ -1,19 +1,25 @@
 package mdoc
 
 import java.util.ServiceLoader
-import mdoc.internal.cli.Settings
+
+import mdoc.internal.cli.{Exit, Settings}
 import metaconfig.ConfDecoder
 import metaconfig.ConfEncoder
 import metaconfig.ConfError
 import metaconfig.generic.Surface
+
 import scala.meta.inputs.Input
 import scala.meta.io.AbsolutePath
 import scala.collection.JavaConverters._
 import scala.meta.io.RelativePath
 
 trait PostModifier {
   val name: String
+  def onStart(settings: Settings): Unit
```

Comment:
These need to have default implementations for backwards compatibility

Reply:
Ok. Will do. Will a simple return of `()` do?

Apologies for the newbie questions. 

I as planning to make another pull request. Should I or do I keep working here? If I do open a new one, their seem to be a "review pull " option. Should I use that instead?

Reply:
`: Unit = ()` is a fine default implementation

Reply:
Done